x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -5,20 +5,26 @@
  * 2.0.
  */
 
-import uuid from 'uuid';
 import {
+  CreateSLO,
+  DefaultResourceInstaller,
+  DefaultTransformInstaller,
   KibanaSavedObjectsSLORepository,
-  ResourceInstaller,
-  TransformInstaller,
 } from '../../services/slo';
 import {
   ApmTransactionDurationTransformGenerator,
   ApmTransactionErrorRateTransformGenerator,
+  TransformGenerator,
 } from '../../services/slo/transform_generators';
-import { SLO } from '../../types/models';
+import { SLITypes } from '../../types/models';
 import { createSLOParamsSchema } from '../../types/schema';
 import { createObservabilityServerRoute } from '../create_observability_server_route';
 
+const transformGenerators: Record<SLITypes, TransformGenerator> = {
+  'slo.apm.transaction_duration': new ApmTransactionDurationTransformGenerator(),
+  'slo.apm.transaction_error_rate': new ApmTransactionErrorRateTransformGenerator(),
+};
+
```

Comment:
💬 We can extract this somewhere when we start reusing it across different routes/services